health/health.c

```diff
@@ -318,6 +317,13 @@ static inline void health_process_notifications(RRDHOST *host, ALARM_ENTRY *ae)
     health_alarm_execute(host, ae);
 }
 
+/**
```

Comment:
Move this comment to another PR

Reply:
I removed, I will create a specific for comments on alarm code.